cypress-io/cypress
# chore: Intregrate runner packages.

- Related with #21689

### User facing changelog

N/A. It's an internal code cleanup.

### Additional details
- Why was this change necessary? => These are mostly-unused legacy code. 
- What is affected by this change? => N/A
- Any implementation details to explain? => I migrated `runner-shared` and `runner-ct` into `runner` and removed unused `scss` files and `react` components.

### Steps to test

`packages` folder structure has been changed. Run all tests to make sure that nothing has changed. 

### How has the user experience changed?

N/A. New contributors might not be confused with the legacy codes.

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
